docs/source/resources/committing.rst

```diff
@@ -0,0 +1,15 @@
+============================
+Committing to the Repository
+============================
+
+1. Make sure you followed the :doc:`Installation Guide </source/installation>`
+2. Go to ClickUp.com
+    * If you are not already signed up, navigate to my.robojackets.com, go to the profile and add an email to the ClickUp field.
+3. Find the issues assigned to you
+4. Move the task to "In Progress" when you start working on it
+5. Create a branch name of the type: **feat/short_issue_title**
+    * Give your issue some good name that is not too long
+6. Once you finish your issue, create a pull request: https://github.com/RoboJackets/roboracing-software/pulls
```

Comment:
Should we require a link to the clickup issue as part of the pull request description? Probably would make it easier to review it does everything its suppose to

Reply:
Yes I like that